packages/api/src/modules/providers/subventia/validators/subventia.validator.test.ts

```diff
@@ -32,174 +32,295 @@ const INVALID_DATA_ROW = {
+                .spyOn(SubventiaValidator, "formatInvalids")
+                .mockReturnValue([
+                    {
+                        ...INVALID_DATA_ROW,
+                        field: "Montant Ttc",
+                        value: "invalidNumber",
+                        message: "Montant Ttc is not a number",
+                    },
+                ]);
         });
 
         afterAll(() => {
-            mockIsDataRowTypesValid.mockRestore();
-            mockIsDataRowCoherenceValid.mockRestore();
+            mockIsSubventiaDtoValid.mockRestore();
+            mockFormatInvalids.mockRestore();
+        });
+
+        it("should call isSubventiaDtoValid", () => {
+            SubventiaValidator.sortDataByValidity(PARSED_DATA);
+            expect(mockIsSubventiaDtoValid).toHaveBeenCalledTimes(PARSED_DATA.length);
         });
 
-        it("should call isDataRowTypesValid", () => {
+        it("should call formatInvalids", () => {
             SubventiaValidator.sortDataByValidity(PARSED_DATA);
-            expect(mockIsDataRowTypesValid).toHaveBeenCalledTimes(PARSED_DATA.length);
+            expect(mockFormatInvalids).toHaveBeenCalledTimes(1);
         });
 
         it("should return an object with valids and invalids keys", () => {
-            mockIsDataRowTypesValid.mockReturnValueOnce(true);
-            mockIsDataRowTypesValid.mockReturnValueOnce(true);
-            mockIsDataRowTypesValid.mockReturnValueOnce(false);
+            mockIsSubventiaDtoValid.mockReturnValueOnce({ valid: true });
+            mockIsSubventiaDtoValid.mockReturnValueOnce({ valid: true });
+            mockIsSubventiaDtoValid.mockReturnValueOnce({
+                valid: false,
+                problems: [{ field: "Montant Ttc", value: "invalidNumber", message: "Montant Ttc is not a number" }],
+            });
 
-            const expected = { valids: [PARSED_DATA_ROW, PARSED_DATA_ROW_2], invalids: [INVALID_DATA_ROW] };
+            const expected = {
+                valids: [PARSED_DATA_ROW, PARSED_DATA_ROW_2],
+                invalids: [
+                    {
+                        ...INVALID_DATA_ROW,
+                        field: "Montant Ttc",
+                        value: "invalidNumber",
+                        message: "Montant Ttc is not a number",
+                    },
+                ],
+            };
             const actual = SubventiaValidator.sortDataByValidity(PARSED_DATA);
             expect(actual).toEqual(expected);
         });
     });
+
+    describe("formatDate", () => {
+        it("should return '' if ExcelDate is null", () => {
+            // @ts-expect-error: protected
+            expect(SubventiaValidator.formatDate(null)).toEqual("");
+        });
+
+        it("should return a Date object if ExcelDate is not null", async () => {
+            const EXCEL_DATE = "44327";
+            const EXPECTED = "11/05/2021";
+            console.log(EXPECTED);
```

Comment:
```suggestion
            console.log(EXPECTED);
```
clean le log dans les tests

Reply:
yes, merci!